policies/recipes/installrecipe.go

```diff
@@ -37,23 +38,27 @@ func InstallRecipe(ctx context.Context, recipe *osconfigpb.SoftwareRecipe) error
 	}
 	installedRecipe, ok := recipeDB.GetRecipe(recipe.Name)
 	if ok {
-		if (!installedRecipe.Greater(recipe.Version)) &&
-			(recipe.DesiredState == osconfigpb.DesiredState_UPDATED) {
+		logger.Debugf("currently installed version of software recipe %s with version %s", recipe.Name, installedRecipe.version)
```

Comment:
Log lines should start with a capitol and end with a period

Reply:
will update